mailbox/backup/src/main/java/org/apache/james/mailbox/backup/Zipper.java

```diff
@@ -37,15 +39,25 @@ public Zipper() {
     }
 
     @Override
-    public void archive(Stream<MailboxMessage> messages, OutputStream destination) throws IOException {
+    public void archive(List<Mailbox> mailboxes, Stream<MailboxMessage> messages, OutputStream destination) throws IOException {
         try (ZipArchiveOutputStream archiveOutputStream = new ZipArchiveOutputStream(destination)) {
+            for (Mailbox mailbox: mailboxes) {
```

Comment:
I'd like this loop to be inside a storeMailbox method and the following one inside a storeMessages method